### ISSUE TITLE
[1.7.10] Soundless ore mining and sudden drop

### ISSUE THREAD

**Issue by Xireka** [TechReborn/TechReborn#576]
When mining the techreborn ores they all break without a sound or any particles, just disappear into thin air as if they were never there and drop their respective ore.

I believe it might be the cause for the crash at #574.

Disabling the generators in the config of extra utilities resolves the crash mentioned in #574  but the lack of sound/particles when breaking techreborn ores is maybe the cause of that crash.

---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 15.05.16 16:28
Description: Playing level event

java.lang.ArrayIndexOutOfBoundsException: 17
at com.rwtema.extrautils.tileentity.generators.BlockGenerator.func_149691_a(BlockGenerator.java:192)
**at net.minecraft.client.particle.EntityDiggingFX.<init>(EntityDiggingFX.java:26)
at net.minecraft.client.particle.EntityDiggingFX.<init>(EntityDiggingFX.java:19)**

Not to mention it's weird when blocks break without particles or a sound.

**Comment by modmuss50** [comment]
Is it just TechReborn blocks? this is VERY strange

**Comment by Xireka** [comment]
Yes only the techreborn ores do that :/

**Comment by modmuss50** [comment]
Mind trying without EXU?

**Comment by Xireka** [comment]
Hi, Since this wasn't cleared yet I did some tests and I wanted to report on what I found out so it's all cleared up.

The crash was weird I mentioned in #574, that came from EXU generators, so I disabled them, loaded my world, re-enabled them and then it worked without crashing.

The soundless ores mining was actually caused by NEID (NotENoughID's mod) every block that had an id over 4399 I believe was soundless. That got fixed in the latest update of the mod so everything is fine now.

Sorry for the late reply on this matter I couldn't test it out earlier. :)
